src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const pricePerDay = 40;
+  const totalAmount = pricePerDay * days;
+  const longTerm = 7;
+  const shortTerm = 3;
+  const longTermDiscount = 50;
+  const shortTermDiscount = 20;
+
+  if (days >= longTerm) {
+    return totalAmount - longTermDiscount;
+  };
```

Comment:
Changed the second if statement to an else if statement. Since both conditions (days >= longTerm and days >= shortTerm) can't be true at the same time, it is more efficient to use else if to avoid unnecessary checks.